src/robot.js

```diff
@@ -49,7 +50,9 @@ class Robot {
       response: new Middleware(this),
       receive: new Middleware(this)
     }
-    this.logger = new Log(process.env.HUBOT_LOG_LEVEL || 'info')
+    process.env.LOG_LEVEL = process.env.LOG_LEVEL || process.env.HUBOT_LOG_LEVEL || 'notice'
```

Comment:
Is default level change intentional (`info` -> `notice`)?

Reply:
Yes. I saw on `log`'s [README.md](https://github.com/medikoo/log#readme)

> LOG_LEVEL
(defaults to notice) Lowest log level from which (upwards) all logs will be exposed.

And so my thinking was to just set it to the default. What do you think?

Reply:
The `info` logs won't be printed if users don't set the env var. I guess not big deal, but probably worth mentioning in release notes

Reply:
I'll change it to default to `info`.